### PR TITLE
Release notes for 8.0.0-beta1

### PR DESCRIPTION
**Pull request by jakelandis**

### CODE REVIEW

**Comment by jrodewig** [comment]
@lockewritesdocs is on the docs rota this week so I've re-assigned him as reviewer. Thanks @jakelandis, @lockewritesdocs!

**Comment by lockewritesdocs** [comment]
@elasticmachine run docs

**Comment by lockewritesdocs** [comment]
@elasticmachine test this please

**Review by lockewritesdocs — APPROVED** [review]
LGTM 👍

File: docs/src/reference/asciidoc/appendix/release-notes/8.0.0-beta1.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[[eshadoop-8.0.0-beta1]]
+== Elasticsearch for Apache Hadoop version 8.0.0-beta1
+
```
Comment (lockewritesdocs) [inline docs/src/reference/asciidoc/appendix/release-notes/8.0.0-beta1.adoc:3]:
```suggestion
coming::[8.0.0-beta1]
```

**Pull request merged**